tools/internal_ci/linux/grpc_e2e_performance_v2.sh

```diff
@@ -37,17 +37,18 @@ tools/run_tests/performance/loadtest_config.py -l go \
     -s big_query_table=e2e_benchmarks.experimental_results \
     -s timeout_seconds=900 --prefix="kokoro-test" -u "$(date +%Y%m%d%H%M%S)" \
     -r go_generic_sync_streaming_ping_pong_secure -o ./loadtest.yaml
+    
+# Dump the contents of the loadtest.yaml (since loadtest_config.py doesn't
+# list the scenarios that will be run).
+cat ./loadtest.yaml
```

Comment:
This is going to be a very long file when more scenarios are included. I suggest instead something like this:

grep '    scenarios: ' ./loadtest.yaml

Reply:
Actually with a '^' at the start of the pattern too, grep '^    scenarios: ' ./loadtest.yaml.

Reply:
The suggestion is fair, but for now (when we have just 1 scenario), the simple `cat` works just fine. let's merge as is and we can always improve once this becomes a problem.

Reply:
It did catch a formatting issue, fixed in #26071. Thanks!